src/lib.rs

```diff
@@ -41,24 +41,24 @@ bitflags::bitflags! {
     pub struct CharacterSet: u8 {
         const Uppercase = 0b0001;
         const Lowercase = 0b0010;
-        const Numbers   = 0b0100;
+        const Digits   = 0b0100;
```

Comment:
nit: please keep aligned
```suggestion
        const Digits    = 0b0100;
```

Reply:
Done!